bin/nvm_install

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
In most recent scripts, we've been adding the `e` flag to fail the script if a command fails and the `u` flag to fail the script if there's an unassigned variable usage. This script defines no variables, but it might in the future.

```suggestion
#!/bin/bash -eu
```

Reply:
> In most recent scripts, we've been adding the `e` flag to fail the script if a command fails and the `u` flag to fail the script if there's an unassigned variable usage. This script defines no variables, but it might in the future.

Thanks @mokagio , this is indeed a great suggestion!